test/manual/specs/recordset.spec.js

```diff
@@ -0,0 +1,21 @@
+var chaisePage = require('./../../e2e/utils/chaise.page.js');
+
+describe('View recordset,', function () {
+    var waitTime = {
+        small: 15000,
+        medium: 30000,
+        large: 120000
+    }
+    beforeAll(function () {
+        browser.ignoreSynchronization = true;
+        browser.get(browser.params.url + "/recordset/#" + browser.params.catalogId + "/product-recordset:accommodation");
+
+        chaisePage.waitForElement(element(by.id("divRecordSet")));
+        chaisePage.recordsetPage.waitForAggregates();
+    });
+
+    // test to setup recordset table with long tooltip text and check the placement of tooltips manually
+    it("should have correct placement of longer tooltip texts on column headers", function () {
+        browser.sleep(waitTime.large);
```

Comment:
I think it's better to use `browser.pause()` for this so we aren't constrained to a certain time limit.

Reply:
Browser.pause() is used if we have an action ( like click or sendkeys) in the test script following this statement (http://www.protractortest.org/#/api?view=ProtractorBrowser.prototype.pause). Since in this script, there is no further action, browser.pause() doesn't stop the execution and the test goes through without waiting.

Reply:
oh, interesting. I hadn't thought of that. I still think pause is better, so maybe we could add `expect(true).toBeTruthy()` and it will behave how we expect it should.

Reply:
My concern with sleep is that you can finish the verification rather quickly and have to wait for that timer to finish before the next `it()` get's called (when we add another `it()` to this spec for other manual tests.